packages/SwingSet/src/controller.js

```diff
@@ -199,13 +203,18 @@ function realmRegisterEndOfCrank(fn) {
     { registerEndOfCrank },
   );
 
-  return (src, filePrefix = '/SwingSet-bundled-source') => {
+  return (src, tag = 'anonymous') => {
+    const filePrefix = `/SwingSet/${tag}`;
+    const localLog = anylogger(`SwingSet:${tag}`);
+
     const nestedEvaluate = source =>
       s.evaluate(source, {
         // Support both getExport and nestedEvaluate module format.
         require: r,
         nestedEvaluate,
 
+        console: harden(localLog),
```

Comment:
Can we audit (or at least skim) the `anylogger` code to make sure this doesn't provide a communication channel between otherwise isolated objects in the vat? We might need to put a wrapper, ala `makeConsole.js`, around the `localLog` to prevent access to mutable state which could be used as a channel.

Reply:
I've audited it (and it's tiny).  Any communications channel is only that which is provided by the application.  The `anylogger-agoric.js` files don't provide anything to an instantiated `log` except access to a subset of `console`.